src/models/tools/shared-model-document-manager.ts

```diff
@@ -86,6 +92,23 @@ types.model("SharedModelDocumentManager")
         return;
       }
 
+      // assign an indexOfType if necessary
+      if (sharedModel.indexOfType < 0) {
```

Comment:
This doesn't seem to be used anywhere. Is it necessary?

Reply:
Table/geometry links are color-coded globally based on their order, so I was planning to use these indices to assign colors. We just haven't gotten there yet.

Reply:
I tried hooking this into the colors for a bit today, but was having trouble accessing it where I needed it, so I left in a couple of notes and am moving on for the time being.